<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWindow</class>
 <widget class="QMainWindow" name="ViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>799</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="CameraControlWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>352</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>352</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="ParaSettingsWidget" name="paraSettingWidget">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="CameraListWidget" name="cameraListWidget"/>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modeSwitchButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Parameter Settings</string>
         </property>
         <property name="icon">
          <iconset resource="3dviewer.qrc">
           <normaloff>:/resources/double_arrow_up.png</normaloff>:/resources/double_arrow_up.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="renderWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="display2d">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <attribute name="title">
           <string>2D</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="render2dTopLayout"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="render2dBottomLayout"/>
           </item>
          </layout>
         </widget>
         <widget class="RenderWidget3D" name="renderWidget3D">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>3D</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGithub"/>
    <addaction name="actionWebSite"/>
    <addaction name="actionOpenLogDir"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionRestartCamera"/>
    <addaction name="actionInfomation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>appExit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionWebSite">
   <property name="text">
    <string>Web Site</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionInfomation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionRestartCamera">
   <property name="text">
    <string>Restart camera</string>
   </property>
  </action>
  <action name="actionOpenLogDir">
   <property name="text">
    <string>Open log directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CameraListWidget</class>
   <extends>QWidget</extends>
   <header>cameralistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParaSettingsWidget</class>
   <extends>QWidget</extends>
   <header>parasettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RenderWidget3D</class>
   <extends>QWidget</extends>
   <header>renderwidget3d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="3dviewer.qrc"/>
 </resources>
 <connections/>
</ui>
